configure.ts

```diff
@@ -47,6 +69,7 @@ export async function configure(command: ConfigureCommand) {
    * Register provider
    */
   await codemods.updateRcFile((rcFile) => {
-    rcFile.addProvider('@neighbourhoodie/adonis-kafka/kafka_provider')
+    rcFile.addProvider(`${command.name}/kafka_provider`)
+    rcFile.addPreloadFile(`#start/kafka`)
```

Comment:
In case the package name changes, use the name from the command.

Add a preload file for `#start/kafka` which is created via stub.